packages/codegen-ui/lib/generate-form-definition/generate-form-definition.ts

```diff
@@ -39,15 +39,19 @@ export function generateFormDefinition({
     elementMatrix: [],
   };
 
-  const modelFieldsConfigs: ModelFieldsConfigs = {};
+  let modelFieldsConfigs: ModelFieldsConfigs = {};
```

Comment:
Why can't this just be declared as `const` on line 50?

Reply:
Line 50 is inside of a conditional so it wouldn't be in scope at line 57.